Backend/Data/PersonRepository.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Backend.Data.Abstraction;
+using Backend.Models;
+using Microsoft.EntityFrameworkCore;
+
+namespace Backend.Data
+{
+    internal class PersonRepository: IPersonRepository
+    {
+        private MyDbContext _context;
+        public PersonRepository(MyDbContext context)
+        {
+            _context = context;
+        }
+
+        public async Task<IList<Person>> GetAllStars()
```

Comment:
Maybe we can make an endpoint exposing something like GetAllStarsForMovie(long id) or GetDirectorForMovie(long id) where we get some information from the PersonRepository